Hunting Queries/SigninLogs/SuccessfulSigninFromNon-CompliantDevice.yaml

```diff
@@ -16,6 +22,35 @@ query: |
   SigninLogs
     | where ResultType == 0
     | where tostring(DeviceDetail.isCompliant) == "false"
+    | join kind=leftouter (
+        IdentityInfo
+        | summarize LatestReportTime = arg_max(TimeGenerated, *) by AccountUPN
+        | extend BlastRadiusInt = iif(BlastRadius == "High", 1, 0)
+        | project AccountUPN, Tags, JobTitle, GroupMembership, AssignedRoles, UserType, IsAccountEnabled, BlastRadiusInt
+        | summarize
+            Tags = make_set(Tags),
+            GroupMembership = make_set(GroupMembership),
```

Comment:
its good practice with make_set to also include the MaxSize value - you've done this in some places but not everywhere.

Reply:
Added!